admin/templates/admin/admin.html

```diff
@@ -21,12 +25,13 @@ <h2 class="card-title">Set Maintenance Banner</h2>
                         <div class="mb-3">
                             <label for="banner">Banner Details:</label>
                             <textarea class="form-control" name="banner" id="banner" rows="2"
-                                placeholder="Enter system maintenance details..."
+                                placeholder="Enter system maintenance details..." style="width: 110%;"
```

Comment:
If possible use the Bootstrap grid system instead of inline CSS

Reply:
Yeah, I intend to refactoring UI designs of the Admin page as the next story, to make feature elements more consistent in visuals. like using one Card for 3 sections and arrange buttons etc. 
This will not mess up this PR. For now, I just removed the inline CSS.